src/app/state/movies/movies-state.ts

```diff
@@ -1,8 +1,10 @@
 import { Movie } from './../../interfaces';
 
 export interface MovieState {
+  loading: boolean;
   movies: Movie[];
 }
 export const moviesInitialState: MovieState = {
+  loading: true,
```

Comment:
a jeśli już byś ustawiał loading dla calego statu, to dlaczego miałby być true domyślnie? Loading robisz jak np pobierasz coś i chcesz pokazać userowi, że się pobiera i czekamy. Nie ma sensu tego dawać jako domyślnego stanu, skąd wiesz kiedy zaczniesz coś pobierać